lib/stat_tracker.rb

```diff
@@ -25,12 +25,32 @@ def self.from_csv(locations)
 
   # ----------SeasonStats
 
+  def find_game_ids_for_season(season)
```

Comment:
Not necessarily a thought to fix during this PR, but wanted to jot down the idea of these 'traffic direction' methods being organized below the main methods the stat tracker needs?